README.md

```diff
@@ -138,6 +120,38 @@ Notable features:
 [^5b]: https://www.ubicloud.com/blog/ubicloud-hosted-arm-runners-100x-better-price-performance
 [^5c]: https://github.com/ubicloud/ubicloud/blob/main/routes/web/webhook/github.rb
 
+### RunsOn
+
+[RunsOn](https://runs-on.com/) is a self-hosted GitHub Actions runner tool that runs on your own AWS account. It's **free for non-commercial usage**, and comes at a €300 flat yearly fee for commercial projects.
+
+Pricing [here](https://runs-on.com/).
+
+Notable features:
+- **Easy install with a QuickFormation template** (video guide available).
+- Cheap, flat-rate pricing with an easy-to-use [pricing calculator](https://runs-on.com/calculator/) that you can use to calculate your spending.
+- ARM & GPU support.
+- Up to 256 vCPUs per runner, by far the most of any provider.
+- Although actively maintained and developed as of Feb. '24, it's built by just one developer ([Cyril Rohr](https://runs-on.com/about/)), so long-term maintenance is a concern. ⚠️
```

Comment:
(The code is public though). But yes I need to work a bit on the license terms to allow for modifications by the buyer when a license has been bought.

Reply:
Nice. In the meantime I added a note to highlight the fact that it's open source. The bus factor is a factor that I'm taking into consideration for all providers though, so while last, I think it's reasonable to keep it in.